bricks/net/tcp/impl/posix.h

```diff
@@ -380,6 +380,25 @@ class ReserveLocalPortImpl final {
       hint, nagle_algorithm_policy, max_connections);
 }
 
+// Acquires the local port with the explicltly provided number, or throws the exception.
```

Comment:
typo: `s/explicltly/explicitly/`

Reply:
Fixed in https://github.com/C5T/Current/pull/990, merging as soon as the tests turn green.